internal/loader_test.go

```diff
@@ -0,0 +1,78 @@
+package internal
+
+import (
+	"fmt"
+	"testing"
+)
+
+func Test_setIndexesToTables(t *testing.T) {
```

Comment:
should I remove this test because exec only test directory now?  (https://github.com/cloudspannerecosystem/yo/blob/master/Makefile#L35)

or change test directory `test` to `./...` 

Which should I do ?